x-pack/plugins/security_solution/public/cloud_security_posture/components/misconfiguration/misconfiguration_preview.tsx

```diff
@@ -144,6 +118,24 @@ export const MisconfigurationsPreview = ({
   const { euiTheme } = useEuiTheme();
   const hasMisconfigurationFindings = passedFindings > 0 || failedFindings > 0;
 
+  const { data: vulnerabilitiesData } = useVulnerabilitiesPreview({
+    query: buildEntityFlyoutPreviewQuery('host.name', name),
+    sort: [],
+    enabled: true,
+    pageSize: 1,
+  });
+
+  const {
+    CRITICAL = 0,
+    HIGH = 0,
+    MEDIUM = 0,
+    LOW = 0,
+    UNKNOWN = 0,
+  } = vulnerabilitiesData?.count || {};
+
+  const totalVulnerabilities = CRITICAL + HIGH + MEDIUM + LOW + UNKNOWN;
+  const hasVulnerabilitiesFindings = totalVulnerabilities > 0;
+
```

Comment:
Is there a nice place to put this into a shared function for here and the same in code fragment from index.tsx? :D